packages/@n8n/nodes-langchain/nodes/output_parser/OutputParserAutofixing/OutputParserAutofixing.node.ts

```diff
@@ -77,8 +97,20 @@ export class OutputParserAutofixing implements INodeType {
 			NodeConnectionType.AiOutputParser,
 			itemIndex,
 		)) as N8nStructuredOutputParser;
+		const prompt = this.getNodeParameter('options.prompt', itemIndex, NAIVE_FIX_PROMPT) as string;
 
-		const parser = new N8nOutputFixingParser(this, model, outputParser);
+		if (prompt.length === 0 || !prompt.includes('{error}')) {
```

Comment:
Are the other placeholders (instructions, completion) not required?

Reply:
I wasn't sure how much we should restrict this; perhaps users would use it with fully custom prompts that provided the completion from other parts of the workflow. But of course, with the default prompt, providing the other two placeholders is semi-required, so it'll work more reliably.

Reply:
Well, it doesn't error, apparently, but it does not yield good results, it looks like (removed the "instructions" bit, but that left an empty object, instead of something that actually matches the output parser...)

Reply:
Good idea. Did that + added more descriptive description 
![CleanShot 2024-11-06 at 12 10 09](https://github.com/user-attachments/assets/6dbdc42b-de17-4247-a1c1-09d00c33af08)